tools/nnyeah/integration/Makefile

```
@@ -16,13 +16,12 @@ TEST_RUN_TARGETS += run-$(1)
 run-$(1): $(1)/bin/Debug/$(1)Integration.dll
 	$(Q) rm -rf Converted/$(1)
 	$(Q) mkdir -p Converted/$(1)
-	-$(Q) $(SYSTEM_DOTNET) run --project ../nnyeah/nnyeah.csproj --input=$(1)/bin/Debug/$(1)Integration.dll --output=Converted/$(1)/$(1)Integration.dll
+	-$(Q) $(SYSTEM_DOTNET) run --project ../nnyeah/nnyeah.csproj --input=$(1)/bin/Debug/$(1)Integration.dll --output=Converted/$(1)/$(1)Integration.dll --xamarin-assembly=$(2) --microsoft-assembly=$(3)
 
 endef
 
-$(eval $(call INTEGRATION_TEST_template,macOS))
-$(eval $(call INTEGRATION_TEST_template,iOS))
-$(eval $(call INTEGRATION_TEST_template,tvOS))
+$(eval $(call INTEGRATION_TEST_template,macOS,../../../_mac-build/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono/Xamarin.Mac/Xamarin.Mac.dll,../../../_build/Microsoft.macOS.Ref/ref/net6.0/Microsoft.macOS.dll))
+$(eval $(call INTEGRATION_TEST_template,iOS,../../../_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.iOS/Xamarin.iOS.dll,../../../_build/Microsoft.iOS.Ref/ref/net6.0/Microsoft.iOS.dll))
```

Use thse variables instead of navigation the folders, this way it'll work when testing against system-installed packages (like we do on our bots now):

```suggestion
$(eval $(call INTEGRATION_TEST_template,macOS,$(MAC_DESTDIR)/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono/Xamarin.Mac/Xamarin.Mac.dll,$(DOTNET_DESTDIR)/Microsoft.macOS.Ref/ref/$(DOTNET_TFM)/Microsoft.macOS.dll))
$(eval $(call INTEGRATION_TEST_template,iOS,$(IOS_DESTDIR)/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.iOS/Xamarin.iOS.dll,$(DOTNET_DESTDIR)/Microsoft.iOS.Ref/ref/$(DOTNET_TFM)/Microsoft.iOS.dll))
```

---

You are correct, I copy pasted strings to see if it worked and forgot to go back and fix them.